.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "i18n-ally.localesPaths": [
+    "src/locales"
+  ],
+  "i18n-ally.keystyle": "nested"
+}
```

Comment:
This should probably be git-ignored?

Reply:
Could be lol, before I commit it :P, but I guess putting it here is somewhat useful when editing the project on another machine.